python/ray/experimental/sgd/examples/tensorflow_train_example.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+from ray import tune
+from ray.experimental.sgd.tensorflow.tensorflow_trainer import (
+    TensorFlowTrainer, TensorFlowTrainable)
+
+from ray.experimental.sgd.tests.tf_helper import (get_model, get_dataset)
```

Comment:
For this, I think we should not use `tf_helper` and define `get_model` and `get_dataset` in this file. PyTorch should be updated in a separate PR.